src/Pagination/Pagination.tsx

```diff
@@ -37,6 +38,8 @@ const Page = styled.a`
     transition-duration: 0.1s;
   }
 
+  ${getGlobalFocusStyles(0)};
```

Comment:
The focus ring on pages has a subtle gray border that makes it look a little "fuzzy":
<img width="143" alt="CleanShot 2022-05-04 at 07 28 25@2x" src="https://user-images.githubusercontent.com/4608155/166703194-690522f7-7634-4055-bda3-e08eb6ec77e5.png">

Reply:
This is another thing I was keeping from Primer CSS.